blocks/api/serializer.js

```diff
@@ -62,7 +63,12 @@ export function getSaveContent( blockType, attributes ) {
 	const contentWithExtraProps = Children.map( saveContent, addExtraContainerProps );
 
 	// Otherwise, infer as element
-	return renderToString( contentWithExtraProps );
+	let content = renderToString( contentWithExtraProps );
+
+	// Drop raw HTML wrappers (support dangerous inner HTML without wrapper)
+	content = content.replace( /<\/?wp-raw-html>/g, '' );
```

Comment:
Is it better to drop after rendering to string? What can be included in `wp-raw-html`?

Reply:
>Is it better to drop after rendering to string? What can be included in `wp-raw-html`?

This is a hacky workaround to the [lack of `<React.Fragment dangerouslySetInnerHTML />` support](https://wordpress.slack.com/archives/C02QB2JS7/p1512663464000044) in React. It achieves the desired result by rendering a wrapper with a known name which is intended to be dropped during serialization. It would be used in cases where we want to render raw markup without a wrapper element (e.g. here for nested blocks, in #3048 for the saved output of Editable).